views/js/runner/testRunner.js

```diff
@@ -82,9 +82,15 @@ define([
                     done(null, $itemContainer);
                 })
                 .assets([
+                    assetStrategies.taomedia,
                     assetStrategies.external,
-                    assetStrategies.base64,
-                    assetStrategies.baseUrl
+                    {name : 'assetHandler', handle : function(url){
```

Comment:
please 
 - reformat, 
 - add a more explicit name like that says "handle assets that comes wihin the package"
 - name your function